packages/core/admin/admin/src/components/GuidedTour/Homepage.tsx

```diff
@@ -51,9 +65,10 @@ const GuidedTourHomepage = () => {
     >
       <Flex direction="column" alignItems="stretch" gap={6}>
         <Typography variant="beta" as="h2">
+          {Object.keys(layoutCopy).length}
           {formatMessage({
             id: 'app.components.GuidedTour.title',
-            defaultMessage: '3 steps to get started',
+            defaultMessage: 'steps to get started',
```

Comment:
```suggestion
            defaultMessage: '{count} steps to get started',
```

You then pass a second argument to `formatMessage` with the object:

```ts
{
	count: Object.keys(layoutCopy).length
}
```